flatliners/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .stddevcluster import StdDevCluster
 from .stddevversion import StdDevVersion
 from .versionedmetrics import VersionedMetrics
+from .alertcorr import AlertCorrelation
```

Comment:
the name of the file should be the same as the class it contains.
so rename the file to `alertcorrelation.py`